### PR TITLE
Update local installations to include pip install of requirements

### PR DESCRIPTION
**Pull request by sgibson91**
This PR updates the installation instructions to include `pip install -r requirements.txt` so that they are complete for running the tutorial notebooks.

Another way of achieving this would be including the contents of `requirements.txt` as a `pip` block in `environment.yaml`, making `requirements.txt` redundant. (**cc:** @tallamjr)

Related issue: https://github.com/devitocodes/devito/issues/1065

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,6 +32,7 @@ git clone https://github.com/devitocodes/devito.git
 cd devito
 conda env create -f environment.yml
 source activate devito
+pip install -r requirements.txt
```
Comment (mloubout) [inline README.md]: That is not necessary, the environment.yml contains all the requirements and installs it

Comment (sgibson91) [reply README.md]: Thanks @mloubout we're having a discussion here: https://github.com/devitocodes/devito/issues/1065

**Comment by navjotk** [comment]
It seems to me that this might not be required eventually. What do you think @sgibson91 ?

**Comment by sgibson91** [comment]
@navjotk Agreed :)

**Pull request closed without merging**